runtime/lua/vim/filetype/detect.lua

```diff
@@ -1146,7 +1146,9 @@ function M.proto(_, bufnr)
   -- comments, thus we can't match comments to see the difference.
   -- IDL files can have a single ';' in the second line, require at least one
   -- character before the ';'.
-  if getline(bufnr, 2):find('.;$') then
+  if getline(bufnr, 2):find('/%* Generated automatically %*/') then
```

Comment:
The comments above should be moved below this condition or removed

Reply:
Oops, missed the comment change.

Reply:
I have to admit I don't understand this patch; the new comment and link don't seem to have anything to do with the condition...